lib/ansible/modules/cloud/opennebula/one_vm.py

```diff
@@ -52,6 +52,9 @@
     description:
       - Password of the user to login into OpenNebula RPC server. If not set
       - then the value of the C(ONE_PASSWORD) environment variable is used.
+      - if either api_username or api_password is not set, then it will try
+      - authenticate with ONE auth file. Default path is "~/.one/one_auth".
+      - Set environment variable C(ONE_AUTH) to override this path.
```

Comment:
You forgot to implement this feature

Reply:
it's not necessary but python-oca will do [that](https://github.com/python-oca/python-oca/blob/0dfd181ee44d4ea05466ca4ed814d290c2c6fcfa/oca/__init__.py#L70) if you pass None to the 'secret' argument.  No need to duplicate the work here.

Reply:
It's not necessary because python-oca will do [that](https://github.com/python-oca/python-oca/blob/0dfd181ee44d4ea05466ca4ed814d290c2c6fcfa/oca/__init__.py#L69) if you pass None as the 'secret' argument. And it's impossible to pass the secret file path to python-oca.